tooling/components/io.siddhi.distribution.editor.core/src/main/resources/web/commons/js/workspace/workspace.js

```diff
@@ -483,6 +484,34 @@ define(['ace/ace', 'jquery', 'lodash', 'log','dialogs','./service-client','welco
                 }
             };
 
+            this.updateExportArtifactsItem = function () {
+                var exportMenuItem = app.menuBar.getMenuItemByID('export.export-for-docker'),
+                    exportKubeMenuItem = app.menuBar.getMenuItemByID('export.export-for-kubernetes');
+
+                var listFilesEndpoint = app.config.services.workspace.endpoint + "/listFiles/workspace?path=" +
+                                                                        app.utils.base64EncodeUnicode("workspace");
+
+                jQuery.ajax({
+                    async: false,
+                    url: listFilesEndpoint,
+                    type: self.HTTP_GET,
+                    success: function (data) {
+                        if (data.length === 0) {
+                            exportMenuItem.disable();
+                            exportKubeMenuItem.disable();
+                        } else {
+                            exportMenuItem.enable();
+                            exportKubeMenuItem.enable();
+                        }
+                    },
+                    error: function (msg) {
+                        console.info("Error getting siddhi apps: " + msg);
```

Comment:
remove console log

Reply:
instead add it to a error popup as discussed

Reply:
Fixed in 21cddd8